### PR TITLE
fix PAE grace notes

### PR DESCRIPTION
**Pull request by rettinghaus**
fix for #642

### CODE REVIEW

File: src/iopae.cpp [hunk 1]
```diff
@@ -92,7 +92,7 @@ bool PaeInput::ImportFile()
     parsePlainAndEasy(infile);
     return true;
 #else
-    LogError("Plain and Easy import is not supported in the build.");
+    LogError("Plaine & Easie import is not supported in the build.");
```
Comment (craigsapp) [inline src/iopae.cpp]: Change in develop branch as well?

File: src/iopae.cpp [hunk 2]
```diff
@@ -104,7 +104,7 @@ bool PaeInput::ImportString(std::string const &pae)
     parsePlainAndEasy(in_stream);
     return true;
 #else
-    LogError("Plain and Easy import is not support in the build.");
+    LogError("Plaine & Easie import is not support in the build.");
```
Comment (craigsapp) [inline src/iopae.cpp]: Change in develop branch as well?

File: src/iopae.cpp (hunk 1 above)
Comment (rettinghaus) [inline src/iopae.cpp]: yes

File: src/iopae.cpp (hunk 2 above)
Comment (rettinghaus) [inline src/iopae.cpp]: yes

**Pull request merged**